packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -291,13 +291,13 @@ export function usePress(props: PressHookProps): PressResult {
         e.stopPropagation();
 
         state.isPressed = false;
-        let target = e.target as HTMLElement;
+        let target = e.target as Element;
         triggerPressEnd(createEvent(state.target, e), 'keyboard', state.target.contains(target));
         removeAllGlobalListeners();
 
         // If the target is a link, trigger the click method to open the URL,
         // but defer triggering pressEnd until onClick event handler.
-        if (state.target.contains(target) && isHTMLAnchorLink(state.target) || state.target.getAttribute('role') === 'link') {
+        if (state.target instanceof HTMLElement && (state.target.contains(target) && isHTMLAnchorLink(state.target) || state.target.getAttribute('role') === 'link')) {
```

Comment:
won't isHTMLAnchorLink be enough to prevent us from running this for an SVG? or why was this changed in this way? I'm not quite following

Reply:
It's because of the below line: only HTMLElements have a `click()` method.

Reply:
ah, typescript stikes again, thanks